docker/docker-executor/app.py

```diff
@@ -0,0 +1,94 @@
+from flask import jsonify, Flask, request
+import os
+import shlex
+import subprocess
+import base64
+
+# Set environment flag of MAX_EXECUTABLE, MAX_DATA_SIZE
+
+
+runtime_version = "google-cloud-function:2.0.3"
+app = Flask(__name__)
```

Comment:
should we try uvicorn and flask?

as it's a single request per container, it should be much different. However, uvicorn is a single-process. it should take less CPU than gunicorn.